packages/backend-plugin-api/src/services/definitions/InstanceMetadataService.ts

```diff
@@ -14,19 +14,15 @@
  * limitations under the License.
  */
 
-/** @alpha */
-export type BackendFeatureMeta =
-  | {
-      type: 'plugin';
-      pluginId: string;
-    }
-  | {
-      type: 'module';
-      pluginId: string;
-      moduleId: string;
-    };
+/** @public */
+export interface InstanceMetadataServicePluginInfo {
+  pluginId: string;
+  modules: {
+    moduleId: string;
+  }[];
+}
 
-/** @alpha */
+/** @public */
 export interface InstanceMetadataService {
-  getInstalledFeatures: () => BackendFeatureMeta[];
+  getInstalledPlugins: () => readonly InstanceMetadataServicePluginInfo[];
```

Comment:
Just double checking for the maintainers - are we safe with leaving this as a non-promise? Forever certain that we never ever want to do anything async in here? :)